extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -95,15 +127,17 @@ public boolean isAnyPropertySet() {
     public static class HibernateOrmConfigQuery {
 
         /**
-         * The max size of the query plan cache.
+         * The maximum size of the query plan cache.
          */
         @ConfigItem
         public Optional<String> queryPlanCacheMaxSize;
 
         /**
-         * The default ordering of nulls specific in the ORDER BY clause.
-         * <p>
-         * Valid values are: none, first, last.
+         * Default precedence of null values in `ORDER BY` clauses.
+         *
+         * Valid values are: `none`, `first`, `last`.
```

Comment:
Shouldn't we replace this `String` type with an enum?

Reply:
Sure, see the discussion with Manyanda above. But I need to check if we can use an ORM one or if we have to create a new one so I consider it outside of the scope of this PR whose main purpose is to validate the Asciidoc support.

Reply:
Ah sorry I didn't see it. Sure it can come in another PR.